packages/babel-plugin-transform-classes/src/transformClass.ts

```diff
@@ -600,8 +602,26 @@ export default function transformClass(
     );
   }
 
+  function extractDynamicKeys() {
+    const { dynamicKeys, node, scope } = classState as {
+      dynamicKeys: Map<string, t.Expression>;
+      node: t.Class;
+      scope: NodePath["scope"];
+    };
+
+    node.body.body.forEach(elem => {
+      if (!t.isClassMethod(elem) || !elem.computed) return;
```

Comment:
Q: Why should we exclude non-method elements? I think we are preserving the order of `ClassElementName` evaluation and thus we should apply it to any named class elements.

Reply:
Yeah, this plugin doesn't support class fields.